lib/workers/pr/index.ts

```diff
@@ -42,6 +42,14 @@ function filterUnavailableUsers(
     : Promise.resolve(users);
 }
 
+function prepareAssigneesReviewers(
+  config: RenovateConfig,
+  usernames: string[]
+): Promise<string[]> {
+  const normalizedUsernames = [...new Set(usernames.map(noLeadingAtSymbol))];
+  return filterUnavailableUsers(config, normalizedUsernames);
```

Comment:
That's the important change: _First_ normalize usernames, _then_ check availability.